docs/source/hardware.rst

```diff
@@ -176,15 +176,19 @@ with the following additional methods and attributes.
     :members:
     :show-inheritance:
 
-    .. attribute:: position
+Certain plans like :func:`~bluesky.plan_stubs.mvr` would like to know where a
+Device was last requested to move to, and other plans like
+:func:`~bluesky.plan_stubs.rd` would like to know where a Device is currently
+located. Devices may implement ``locate()`` to provide this information.
 
-        A heuristic that describes the current position of a device as a
-        single scalar, as opposed to the potentially multi-valued description
-        provided by ``read()``.
+.. autoclass:: bluesky.protocols.Locatable
+    :members:
+    :show-inheritance:
 
-        Optional: bluesky itself does not use the position attribute, but other
-        parts of the ecosystem might.
-        Developers are encouraged to implement this attribute where possible.
```

Comment:
It might make sense to keep the `position` attribute in the documentation in some sort of deprecated sense to help people who stumble upon old devices that have not implemented `locate`.

Reply:
Agreed.